airflow-core/src/airflow/config_templates/airflow_local_settings.py

```diff
@@ -129,7 +129,16 @@
 ##################
 
 REMOTE_LOGGING: bool = conf.getboolean("logging", "remote_logging")
-REMOTE_TASK_LOG: RemoteLogIO | None = None
+REMOTE_TASK_LOG: (
+    RemoteLogIO
+    | S3RemoteLogIO
+    | CloudWatchRemoteLogIO
+    | GCSRemoteLogIO
+    | WasbRemoteLogIO
+    | OSSRemoteLogIO
+    | HdfsRemoteLogIO
+    | None
+) = None
```

Comment:
This shouldn't be required -- RemoteLogIO is a protocol that say what the types need to implement